### ISSUE TITLE
Media saved by ... (receipt removal)

### ISSUE THREAD

**Issue by Sepero** [oxen-io/session-android#602]
A recent addition to the latest version of Session is notification of Media saved by the receiver in 1 to 1 chats.

In my humble opinion, the app shouldn't be sending this information, but please let me explain reasons why.

1. If I send a message, then that was purposeful/intentional.
2. If I send a file, then that was intentional.
3. If I send a link with a preview, that I send is intentional, and under my control.

When I save a image or file, I do not intend to send data across the network. Letting others know that the file was saved is not essential information to our communications. In secure communications, I should think it undesirable to unintentionally/involuntarily be sending nonessential information across the network.

Also, if someone wanted to, they could modify the source code, and delete the code sending information. They could save all media, and the sender wouldn't know. Therefore, if a image or file is sent, we should assume by default that it was saved.

TLDR; 1) Communicating this information is not essential. 2) Secure communication should be intentional. 3) Modification of the code means we can never trust whether the message is true or not

My request: In an effort to keep Session a Simple app, I respectfully request that this new feature be removed. If not removed, made into a toggle option in Settings.

Lastly, please don't take offense to this issue. I understand that this is a new feature, and it takes pain staking time to implement such things. We all greatly appreciate all the work and development that goes into Session. Session is loved and very much needed.

PS.
I think this issue highlights a broader issue regarding security, and setting standards for what information should and should not be transmitted. The set of fundamental Standards/Guidelines for the future of Session development.

**Comment by Sepero** [comment]
(Author of Swedish and Thai translations for Session) thank you

**Comment by KeeJef** [comment]
I think this is moving towards our philosophy of silent notifications for actions which could expose the chat participants privacy, like screenshot notifications and media save notifications. But this is perhaps part of a larger discussion.

**Comment by tcely** [comment]
Wouldn't the more appropriate notification be on the sending side?

You sent a file, you no longer have any control over what the recipient / their software does with that file. 😉

**Comment by x80486** [comment]
I think it makes sense to avoid such notifications as well. From the moment something is sent, the sender really lost control (even with the "Delete for everyone" feature) of what really happened on the other side.

Moreover, what can we do with those notifications anyway?
